### ISSUE TITLE
[Feature] Allow disable built-in controller settings and allow setting up with Midnight Controls Mod.

### ISSUE THREAD

**Issue by Fl414r** [PojavLauncherTeam/PojavLauncher#3799]
### Describe the bug

Due to the controller's built-in settings, I can't set it up properly with the Midnight Controls Mod.

### The log file and images/videos

Not a bug

### Steps To Reproduce

```markdown
1) Try setting up the Controller with Midnight Controls
2) Get failed.
```

### Expected Behavior

I expect to be able to set up a controller with Midnight Controls the way I'm used to.

### Platform

```markdown
- Device model: Honor 9C 4/64G (AKA-L29)
- CPU architecture: aarch64
- Android version: 10
- PojavLauncher version: Android CL #4084 (upd. And CL #4087)
```

**Comment by Fl414r** [comment]
<details>
<summary>video without voice</summary>
 
In settings i try change buttons, after I joined server first - I try Run on L3, second - try open F3 by Up+Circle and in the end - I try change Rotation speed but it all not work.

https://user-images.githubusercontent.com/74704475/209453306-11a7d24f-9921-4045-9240-c8cb261914b5.mp4
</details>

**Comment by Fl414r** [comment]
In settings i try change buttons. After I joined server: first - I try Run on L3, second - try open F3 by Up+Circle and in the end - I try change Rotation speed. All not work. All controller settings have been replaced with Pojav settings.
<details>
<summary>video (no voice sorry)</summary>

https://user-images.githubusercontent.com/74704475/209453306-11a7d24f-9921-4045-9240-c8cb261914b5.mp4
</details>
